src/sql/workbench/contrib/editData/browser/editDataGridPanel.ts

```diff
@@ -725,4 +775,328 @@ export class EditDataComponent extends GridParentComponent implements OnInit, On
 		}
 
 	}
+
+	private createNewTable(): void {
+		let newGridContainer = document.createElement('div');
+		newGridContainer.className = 'grid';
+
+		if (this.placeHolderDataSets) {
+			let dataSet = this.placeHolderDataSets[0];
+			let options = {
+				enableCellNavigation: true,
+				enableColumnReorder: false,
+				renderRowWithRange: true,
+				showHeader: true,
+				rowHeight: this.rowHeight,
+				defaultColumnWidth: 120,
+				editable: this.enableEditing,
+				autoEdit: this.enableEditing,
+				enableAddRow: false, // TODO change when we support enableAddRow
+				enableAsyncPostRender: false,
+				editorFactory: {
+					getEditor: (column: ISlickColumn<any>) => this.getColumnEditor(column)
+				},
+				formatterFactory: {
+					getFormatter: this.getFormatter
+				}
+			};
+
+
+			if (dataSet.columnDefinitions) {
+
+				this.tables[0] = new Table(this.nativeElement.appendChild(newGridContainer), { dataProvider: this.gridDataProvider, columns: dataSet.columnDefinitions }, options);
+
+				for (let plugin of this.plugins) {
+					this.tables[0].registerPlugin(plugin);
+				}
+
+				for (let i = 0; i < this.placeHolderDataSets[0].columnDefinitions.length; i++) {
```

Comment:
may consider using `dataSet` here since that is what was checked in the `if` guard condition.  though they appear to always be the same in the case.